s/soap-builder/soap-builder_ubi8.4.sh

```diff
@@ -1,7 +1,7 @@
 # ----------------------------------------------------------------------------
 #
 # Package       : soap-builder
-# Version       : 1.0.0-SNAPSHOT
+# Version       : 93e5a7956ed68b6125ab756e9c6cddea266a9dc6 (commit ID)
```

Comment:
We specify commit SHAs in the following format in general:
```
commit # 93e5a7956ed68b6125ab756e9c6cddea266a9dc6
```
Please use this format in the future.